index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>CV - Pinchuk Vladislav</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+       div {
```

Comment:
I would like to use a class instead of the selector by type. At least because of HTML document could have a few `div` tags in the future.